packages/cli/package.json

```diff
@@ -1,9 +1,36 @@
 {
   "name": "@code-pushup/cli",
   "version": "0.5.0",
+  "license": "MIT",
   "bin": {
     "code-pushup": "index.js"
   },
+  "homepage": "https://github.com/code-pushup/cli#readme",
+  "bugs": {
+    "url": "https://github.com/code-pushup/cli/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/code-pushup/cli.git",
+    "directory": "packages/cli"
+  },
+  "contributors": [
+    {
+      "name": "Matěj Chalk",
+      "email": "matej.chalk@flowup.cz",
+      "url": "https://flowup.cz/en"
+    },
+    {
+      "name": "Michael Hladky",
+      "email": "michael.hladky@push-based.io",
+      "url": "https://push-based.io"
+    },
+    {
+      "name": "Katerina Pilatova",
+      "email": "katerina.pilatova@flowup.cz",
+      "url": "https://flowup.cz/en"
+    }
+  ],
```

Comment:
See comment above. I would move it out and put a contributors section in readme

Reply:
> (Apparently, [NPM autofills contributors if there is an AUTHORS file](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#default-values), could be another alternative.)

Thanks, that's a nice alternative!

Reply:
@matejchalk 

> Also, `MishaSeredenkoPushBased` and `IKatsuba` are missing from this list.

Are they part of the Code PushUp core team? Otherwise, we could use an All Contributors spec to add community contributors.

Reply:
> > Also, MishaSeredenkoPushBased and IKatsuba are missing from this list.
> Are they part of the Code PushUp core team?

Yes, they're both part of the core team as well.

Reply:
@MishaSeredenkoPushBased and @IKatsuba have been added to the `contributors` section of the root `package.json` file and are copied via our `PackageJSON` ESBuild plugin.

Reply:
@BioPhoton 

> I would move it out and put a contributors section in readme

We could use [All Contributors](https://allcontributors.org/) for this but for npm and other tools, we need `package.json` metadata.